### PR TITLE
Email indicator

### PR DESCRIPTION
**Pull request by heron2014**
Related to this #258 

@nelsonic please have a look at this PR, thank you

@iteles UI are in this [comment](https://github.com/FAC-GM/app/issues/258#issuecomment-186189572) , please have a look at them , thank you

### CODE REVIEW

**Comment by SimonLab** [comment]
Ready to be merged!

File: test/emailIndicatorHelper.test.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+});
+
+describe('Pass the last email object with timestamp which is more than 3 months ', function () {
+
+  it('return the icon with the color green ', function (done) {
+
+    var sentEmailObj = {sentAt: '14-02-2016', timestamp: new Date().getTime() - (100 *24 *60 *60 *1000)};
+    var result = emailIndicator(sentEmailObj, true);
+
+    expect(result.string).to.equal("<i class='fa fa-paper-plane last-email-regular'><span class='last-email-regular'> Emailed after 3 months</span></i>");
```
Comment (iteles) [inline test/emailIndicatorHelper.test.js]: The text here should be 'Have not emailed for over 3 months'

Comment (heron2014) [inline test/emailIndicatorHelper.test.js]: @iteles Done. As we agreed we have changed to : Emailed over 3 months ago.

**Comment by iteles** [comment]
@nelsonic I'm testing this locally now :+1:

**Comment by nelsonic** [comment]
Looks good. :+1:

**Pull request merged**